### ISSUE TITLE
tabs_on_bottom_menubar_on_top_patch.css not working (linux, firefox 115)

### ISSUE THREAD

**Issue by doomvox** [MrOtherGuy/firefox-csshacks#309]
I've tried using the tabs_on_bottom_menubar_on_top_patch.css and it leaves my tab bar up above the URL bar, instead of placing it below the bookmarks bar as desired.  

I'm using Firefox 115.3.1esr (64-bit) on Debian linux, version 11 (bullseye).

I use the code in my chrome/userChrome.css like so:
  @import url("../../../lib/tabs_on_bottom_menubar_on_top_patch.css");

(I've been using this approach for over a year now, the relative path is correct.)

**Comment by MrOtherGuy** [comment]
That is simply a patch file. You'll need to first import tabs_on_bottom.css which does the heavy lifting, then import this patch after that.

**Comment by doomvox** [comment]
Thanks, I'll try that.  Hm... still doesn't seem to be working, though now I'm doing this: 

@import url("../../../lib/tabs_on_bottom.css");
@import url("../../../lib/tabs_on_bottom_menubar_on_top_patch.css");

I'm using manually installed files, but I did refresh them recently.

**Comment by MrOtherGuy** [comment]
Yes, that looks correct - at least if the file path leads to correct location.

Note that all import statements need to be before anything else in the userChrome.css file.

Another thing you can try is to put the file content(s) directly into userChrome.css to make sure that works.
